readdir_test.go

```diff
@@ -0,0 +1,95 @@
+package hdfs
+
+import (
+	"fmt"
+	"math/rand"
+	"path"
+	"testing"
+
+	"github.com/beyondstorage/go-endpoint"
+	ps "github.com/beyondstorage/go-storage/v4/pairs"
+	"github.com/beyondstorage/go-storage/v4/types"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHdfsDirerWithFiles(t *testing.T) {
+	for i := 0; i < 100; i++ {
+		numbers := 1000 + rand.Intn(1000)
+
+		t.Run(fmt.Sprintf("list %d files", numbers), func(t *testing.T) {
+			testHdfsReaddir(t, numbers)
+		})
+	}
+}
+
+func testHdfsReaddir(t *testing.T, numbers int) {
```

Comment:
Can you add some comments to explain what is tested beyond https://github.com/beyondstorage/go-integration-test/blob/master/direr.go?

Reply:
Determine if the directory and file names are the same when listing 1000 files each time in 100 loops
I'd prefer to do a stress test, but that's slower

Reply:
Please pass integration tests before implement more strict tests.

Reply:
This looks like` os.Getenv()` doesn't work internally
But if we write dead address nodes will not have this environment
This may have to use [setup-hdfs](https://github.com/beyondstorage/setup-hdfs)
But this is my first time learning CI, may be trial and error